tsl/test/shared/expected/dist_chunk.out

```diff
@@ -0,0 +1,55 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+SELECT chunk_name, data_nodes 
+FROM timescaledb_information.chunks 
+WHERE hypertable_name = 'dist_chunk_copy';
+       chunk_name        |        data_nodes         
+-------------------------+---------------------------
+ _dist_hyper_15_67_chunk | {data_node_1,data_node_2}
+ _dist_hyper_15_68_chunk | {data_node_2,data_node_3}
+ _dist_hyper_15_69_chunk | {data_node_1,data_node_3}
+(3 rows)
+
+SELECT chunk_name 
+FROM timescaledb_information.chunks 
+WHERE hypertable_name = 'conditions';
+     chunk_name     
+--------------------
+ _hyper_10_51_chunk
+ _hyper_10_52_chunk
+(2 rows)
+
+\set ON_ERROR_STOP 0
+SELECT _timescaledb_internal.create_chunk_replica_table(NULL, 'data_node_1');
+ERROR:  chunk cannot be NULL
+SELECT _timescaledb_internal. create_chunk_replica_table('_timescaledb_internal._dist_hyper_15_67_chunk', NULL);
+ERROR:  data node name cannot be NULL
+SELECT _timescaledb_internal.create_chunk_replica_table('metrics_int', 'data_node_1');
+ERROR:  relation "metrics_int" is not a chunk
+SELECT _timescaledb_internal.create_chunk_replica_table('conditions', 'data_node_1');
+ERROR:  relation "conditions" is not a chunk
+SELECT _timescaledb_internal. create_chunk_replica_table('_timescaledb_internal._hyper_10_51_chunk', 'data_node_1');
+ERROR:  chunk "_hyper_10_51_chunk" doesn't belong to a distributed hypertable
+SELECT _timescaledb_internal.create_chunk_replica_table('_timescaledb_internal._dist_hyper_15_67_chunk', 'data_node_1');
+ERROR:  chunk "_dist_hyper_15_67_chunk" already exists on data node "data_node_1"
+SELECT _timescaledb_internal.create_chunk_replica_table('_timescaledb_internal._dist_hyper_15_27_chunk', 'data_node_1');
+ERROR:  relation "_timescaledb_internal._dist_hyper_15_27_chunk" does not exist at character 57
+SELECT _timescaledb_internal.create_chunk_replica_table('_timescaledb_internal._dist_hyper_15_67_chunk', 'data_node_4');
+ERROR:  server "data_node_4" does not exist
+BEGIN READ ONLY;
+SELECT _timescaledb_internal.create_chunk_replica_table('_timescaledb_internal._dist_hyper_15_67_chunk', 'data_node_3');
+ERROR:  cannot execute create_chunk_replica_table() in a read-only transaction
+COMMIT;
+\set ON_ERROR_STOP 1
+SELECT _timescaledb_internal.create_chunk_replica_table('_timescaledb_internal._dist_hyper_15_67_chunk', 'data_node_3');
```

Comment:
It would be good with e.g., a `remote_exec` to see that this chunk table was actually created on the data node.

Reply:
The second call gives an error on the corresponding data node that the relation already exists. Isn't it enough?

Reply:
I guess, although as a reviewer it is non-obvious that the failure is generated by a table conflict on the DN as opposed to some kind of early check on the AN. For instance, the AN could have created some local metadata for the table so it fails on checking for the metadata. But it could still have failed to create the table object on the DN. You'd have to know the implementation details to understand whether this is the case or not. 

IMO, it is mentally easier to follow when there is a follow-up check to show that the object in fact was created.

Reply:
@erimatnor Have you looked to the error?
```
ERROR:  [data_node_3]: relation "_dist_hyper_15_67_chunk" already exists
```
It clearly says that the failure is on the data node.

Reply:
I am giving the feedback from the point of view of a reviewer. And, Yes, reviewers might not always notice or capture every detail, so just letting you know what I "expected to see" and things that are easy to miss otherwise.

Like with code, even if something is technically correct, the behavior isn't always obvious even though all the information is technically there (however subtly).

I approved, so this is certainly not a show-stopper so if you believe it is more effort than it is worth, then I'll let you be the judge of that. Just giving feedback.

Reply:
I added a comment to clarify this in the test.

Reply:
Using `remote_exec` is nontrivial in shared tests. After spending considerable time I fall back to connecting and running the query on the data node.